account_invoice_partner_no_autofollow/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+
+By default Odoo adds partner as follower to all invoices.
+This module should disable this behaviour.
```

Comment:
By default Odoo adds partner as follower to all invoices.
This module allows to disable this behaviour using General Settings.

Reply:
done